src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter.py

```diff
@@ -51,58 +51,65 @@ def test_containerapp_session_code_interpreter_e2e(self, resource_group):
             identifier_name,
             code),
             checks=[
-            JMESPathCheck('properties.status', 'Success'),
-            JMESPathCheck('properties.stdout', 'Hello world\n')
+            JMESPathCheck('status', 'Succeeded'),
+            JMESPathCheck('result.stdout', 'Hello world\n')
         ])
 
         # upload a file also add session pool location
         txt_file = os.path.join(TEST_DIR, 'cert.txt')
-        self.cmd('containerapp session code-interpreter upload-file -n {} -g {} --identifier {} --filepath "{}" --session-pool-location {}'.format(
+        path = '/'
+        self.cmd('containerapp session code-interpreter upload-file -n {} -g {} --identifier {} --filepath "{}" --path {} --session-pool-location {}'.format(
             sessionpool_name_python,
             resource_group,
             identifier_name,
             txt_file,
+            path,
             TEST_LOCATION),
             checks=[
-            JMESPathCheck('value[0].properties.filename', 'cert.txt'),
+            JMESPathCheck('name', 'cert.txt'),
         ])
 
         # list files
-        files_list = self.cmd("containerapp session code-interpreter list-files -n {} -g {} --identifier {}".format(
+        files_list = self.cmd("containerapp session code-interpreter list-files -n {} -g {} --identifier {} --path {}".format(
             sessionpool_name_python,
             resource_group,
-            identifier_name)).get_output_in_json()
+            identifier_name,
+            path)).get_output_in_json()
         self.assertTrue(len(files_list["value"]) == 1)
 
         # check content
-        file_content = self.cmd("containerapp session code-interpreter show-file-content -n {} -g {} --identifier {} --filename {}".format(
+        file_content = self.cmd("containerapp session code-interpreter show-file-content -n {} -g {} --identifier {} --filename {} --path {}".format(
             sessionpool_name_python,
             resource_group,
             identifier_name,
-            "cert.txt")).get_output_in_json()
+            "cert.txt",
+            path)).get_output_in_json()
         self.assertTrue(file_content == '\"testing\"')
 
         # check metadata
-        self.cmd("containerapp session code-interpreter show-file-metadata -n {} -g {} --identifier {} --filename {}".format(
+        self.cmd("containerapp session code-interpreter show-file-metadata -n {} -g {} --identifier {} --filename {} --path {}".format(
```

Comment:
The original test case cover `--filename`. 
For `--path` we should crease a separate test case for it.

Reply:
Thanks, new test case added